LOUPE_Backend/SynchronizationService.API/SyncLogService/SyncLogService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Google.Protobuf;
+using Grpc.Core;
+using Grpc.Net.Client;
+using SynchronizationService.Core.API.ViewModels;
+using SyncronizationService.API;
+using System.Collections.ObjectModel;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace SynchronizationService.API.SyncLogService
+{
+    public class SyncLogService
+    {
+        private readonly GrpcChannel _channel;
+        public SyncLogService()
+        {
+            _channel = GrpcChannel.ForAddress("https://localhost:5101");
+        }
+        
+        public async Task SendTransformationsToLoggingAsync(Collection<TransformationViewModel> transofrmations)
+        {
+            SyncLogRequest request = new SyncLogRequest
+            {
+                UserId = transofrmations[0].UserId.ToString(),
+                GroupId = transofrmations[0].GroupId.ToString(),
+                StartSyncronizationId = transofrmations[0].Id.ToString(),
+                EndSyncronizationId = transofrmations[^1].Id.ToString()
+            };
+
+            SyncLog.SyncLogClient client = new SyncLog.SyncLogClient(_channel);
+            SyncLogReply reply = await client.GenerateLogAsync(request);
+
+            Console.WriteLine(reply.Succeded.ToString());
```

Comment:
Weet niet of console.writelines een methode is waarop we dingen willen loggen. Ik ga er van uit dat dit iets gegenereerd is maar misschien kunnen we hier met een extern logging tool op aanhaken?

Reply:
Lijkt mij een goed plan!